packages/paste-core/components/truncate/src/index.tsx

```diff
@@ -4,6 +4,7 @@ import {Box, BoxProps} from '@twilio-paste/box';
 
 export interface TruncateProps extends Omit<React.HtmlHTMLAttributes<HTMLSpanElement>, 'color'>, Pick<BoxProps, 'as'> {
   children: NonNullable<React.ReactNode>;
+  title: NonNullable<string>;
```

Comment:
nit: you can omit the NonNullable here. https://www.typescriptlang.org/docs/handbook/utility-types.html#nonnullabletype